ddp-workspace/projects/ddp-osteo/src/assets/i18n/en.json

```diff
@@ -60,30 +60,30 @@
             {
                 "id": 0,
                 "title": "Touring the Genomics Platform",
-                "text": "Members of the Count Me In team and the Osteo Project’s Patient Advisory Council tour the Genomics Platform at the Broad Institute.",
-                "image": "assets/images/osteo-team.png",
-                "alt": "Osteo team"
+                "text": "Members of the Count Me In Team and the OSP Project Advisory Council tour the Genomics Platform at the Broad Institute.",
```

Comment:
"OSP Project" does not sound right to me. I thought OSP stood for "Osteosarcoma Project", so OSP Project would not make sense. You got this text from Erin? Maybe she can confirm?

Reply:
[OSP] is one thing, [Project Advisory Council] aka PAC is another thing.